### PR TITLE
Cleanup the Cell Renderer Codebase

### PR DESCRIPTION
**Pull request by andersonjwan**
Hello, this is my first ever contribution to an open-source project, and the prospect of GTG seems very promising. I thought I'd start small with some code formatting cleanup and better self-documentation for readability.

If you have any suggestions, please let me know. I will be more than willing to adjust as necessary. Here's to my first contribution, hopefully! 🎉

The following errors and warnings according to PyCodeStyle have been fixed:

- E402 (import order)
- E261 (comment spacing)
- E303 (too many blank lines)
- W504 (line break after a binary operator)
- E222 (too many spaces after operator)
- E305 (expected 2 blank lines after class declaration)

### CODE REVIEW

File: GTG/gtk/browser/cell_renderer_tags.py [hunk 1]
```diff
@@ -42,24 +43,43 @@ class CellRendererTags(Gtk.CellRenderer):
...
-        context.line_to(x + w, y + h - r)                      # Move to D
+        #   A  *  BQ
+        #  H       C
+        #  *       *
+        #  G       D
+        #   F  *  E
+
+        context.move_to(
+            x + r, y
+        )  # Move to A
```
Comment (Neui) [inline GTG/gtk/browser/cell_renderer_tags.py:54]: I think you wanted to format like the following `curve_to` calls, but it doesn't have many parameters and maybe could easily led to make one think it is a `curve_to` call. It's not too bad tho.

**Review by Neui — APPROVED** [review]
Looks OK and still works (locally tested).

I personally would prefer if such style commits were squashed together into one, since they are usually uninteresting and having many style-commits doesn't really help trying to find behaviour differences.

Also, we have a [`pylint.rc`](https://github.com/getting-things-gnome/gtg/blob/master/pylint.rc) configuration file. I'm not into the ecosystem, but I wonder about the "compatibility" with PyCodeStyle, like does it respect that file? Doesn't seems so [according to the documentation](https://pycodestyle.pycqa.org/en/latest/intro.html#configuration). We should decide on what we should use.
(Also, I need to check if my editor respects it or whatever linter my editor uses...)

**Comment by andersonjwan** [comment]
> I personally would prefer if such style commits were squashed together into one, since they are usually uninteresting and having many style-commits doesn't really help trying to find behaviour differences.

Understood. I wanted to be as clear as possible with my intentions, but from here forward, for more stylistic and formatting changes, I will group into a single commit that covers each error/warning fixed to simplify the review process.

> Also, we have a [`pylint.rc`](https://github.com/getting-things-gnome/gtg/blob/master/pylint.rc) configuration file. I'm not into the ecosystem, but I wonder about the "compatibility" with PyCodeStyle, like does it respect that file? Doesn't seems so [according to the documentation](https://pycodestyle.pycqa.org/en/latest/intro.html#configuration). We should decide on what we should use.

I was originally using the command below (found [here](https://github.com/getting-things-gnome/gtg/blob/master/docs/contributors/coding%20best%20practices.md#checking-the-coding-style) in the docs) to determine the set of formatting errors. I also just went ahead and installed pylint and setup the proper linting to be based off of the `pylint.rc` file, and I do not receive any formatting errors - only argument/variable usage warnings.

```sh
find . -name '*.py' -print0 | xargs -0 pycodestyle --repeat --max-line-length=100 --ignore=E128
```

If there are any further checkers/configurations that I should verify, please let me know. Thank you!

File: GTG/gtk/browser/cell_renderer_tags.py (hunk 1 above)
Comment (andersonjwan) [reply GTG/gtk/browser/cell_renderer_tags.py]: I agree. I've made the changes and pushed them.

**Comment by diegogangl** [comment]
LGTM, thanks! Congrats on your first contribution :beers:

**Pull request merged**